Automattic/simplenote-macos
# Setup saving simperium token to keychain on simperium did login

### Fix
So the next shortcut/intent that we need to add is to be able to update a note from a shortcut.  This is best done using the simperium API so that versioning is handled on the server.  To do that we need to be able to access the api token that we get in the main app on login from the intents extension.  So this PR sets that up.

### Test
1. Fresh install Simplenote mac
2. Log into your account.
3. Go to the file `OpenNoteIntentHandler.swift` and in line 28 add this code:

```
let token = KeychainManager.extensionToken
        print("# \(token)")
```

4. Run the intents extension using the Shortcuts app.  Create a shortcut that opens a note (any note it doesn't matter) and run the shortcut

Confirm that you see the token in the logs


### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> `RELEASE-NOTES.txt` was updated in d3adb3ef with:
> 
> > Added markdown support

If the changes should not be included in release notes, add a statement to this section. For example:

> These changes do not require release notes.
